### ISSUE TITLE
Wrong bulk links generated

### ISSUE THREAD

**Issue by DmiTools** [BranchMetrics/branch-deep-linking-public-api#15]
Hello.

I'm using your branch HTTP API and after 20 june 2016 I found strange bug: 

links from bulk request (http://api.branch.io/v1/url/bulk/{key}) in api response stay wrong, such as http://{our_domain}/Vx6Hidkjru, and they not redirect to $fallback_url, which i'm writing into request. 

Also checking this links with out api (https://api.branch.io/v1/url?url=) return me error 404 "The resource you were looking for could not be found." 

Simple single request with the same parameters generate good links, which redirects correctly (such as http://{our_domain}/Y99d/yawPQnYhru).

Api docs, which i'm using, i found there:  https://github.com/BranchMetrics/branch-deep-linking-public-api

**Comment by aaustin** [comment]
Hey @DmiTools - Apologies for this. We found a bug with our new domains `app.link` that broken all other custom domains as well. We've got a patch on master and hope to deploy it today. Your links were actually created but the CSV returned links without the require `Y99d`. If you want to correct the bulk export, just add that 4 character string in there and you'll be good. Then, after today, any additional bulk links will be returned in the correct format.

Apologies!
